tutorials/plot_background_filtering.py

```diff
@@ -483,15 +524,232 @@ def plot_signal(x, offset):
 plt.show()
 
 ###############################################################################
-# Filtering in MNE-Python
-# =======================
+# Some pitfalls of filtering
+# ==========================
+# Multiple recent papers have noted potential risks of drawing
+# errant inferences due to misapplication of filters.
+#
+# Low-pass problems
+# -----------------
+# Filters in general, especially those that are acausal (zero-phase), can make
+# activity appear to occur earlier or later than it truly did. As
+# mentioned in VanRullen 2011 [3]_, investigations of commonly (at the time)
+# used low-pass filters created artifacts. However, such deleterious effects
```

Comment:
Actually even filtering at 40 Hz was problematic because of their use of `firls` with don't-care regions -- IIRC they designed a really sharp transition band filter, but said the transition band was "don't care", and I suspect the filter design function ended up creating a nasty resonance there. I'll try to find a way to re-word it.